src/emulator/gxm/src/gxm.cpp

```diff
@@ -512,6 +512,24 @@ void flip_vertically(uint32_t *pixels, size_t width, size_t height, size_t strid
     }
 }
 
+SceGxmTextureFormat get_texture_format(SceGxmTexture *texture){
+    return (SceGxmTextureFormat)(texture->controlWords[1] & 0x1F000000 | texture->controlWords[0] & 0x80000000 | HIWORD(texture->controlWords[3]) & 0x7000);
+}
+
+unsigned int get_texture_width(SceGxmTexture *texture){
+    if (texture->controlWords[1] & 0xA0000000){
```

Comment:
0xa is two separate bits set. What do those bits mean? Non power of two? I think this needs to be self-documenting, or documented.